apply_manual_codes/apply_manual_codes.py

```diff
@@ -0,0 +1,135 @@
+import argparse
+import time
+from os import path
+import json
+from dateutil.parser import isoparse
+
+from core_data_modules.cleaners import CharacterCleaner, Codes
+from core_data_modules.cleaners.codes import SomaliaCodes
+from core_data_modules.cleaners.location_tools import SomaliaLocations
+from core_data_modules.traced_data import Metadata
+from core_data_modules.traced_data.io import TracedDataJsonIO, TracedDataCodaIO, TracedDataTheInterfaceIO
+from core_data_modules.util import IOUtils
+
+
+# TODO: remove once pull request for below merged with master in CoreDataModules
```

Comment:
Did you need to edit anything here or is it taken directly?

If no edits were made, you could change the ref in the Pipfile to the git commit hash of the commit in Core Data you need.